dev-libs/libdaemon/libdaemon-0.14-r4.ebuild

```diff
@@ -27,40 +27,36 @@ src_prepare() {
 	# (elibtoolize is insufficient)
 	# bug #668404
 	eautoreconf
-
-	# doxygen is broken with out-of-source builds
-	multilib_copy_sources
 }
 
-multilib_src_configure() {
+src_configure() {
 	econf \
 		--localstatedir=/var \
 		--disable-examples \
-		--disable-lynx \
-		$(use_enable static-libs static)
+		--disable-lynx
 }
 
-multilib_src_compile() {
+src_compile() {
 	emake
 
-	if multilib_is_native_abi && use doc; then
+	if use doc; then
 		einfo "Building documentation"
 		emake doxygen
 	fi
 }
 
-multilib_src_install() {
+src_install() {
 	emake DESTDIR="${D}" install
 
-	if multilib_is_native_abi && use doc; then
+	if use doc; then
 		docinto html
 		dodoc -r doc/README.html doc/style.css doc/reference/html/*
```

Comment:
put this is into `src_compile` and set the `HTML_DOCS` there

Reply:
Can you even call dodoc in src_compile? It involves knowledge of ${ED}?

Reply:
what I mean was do `HTML_DOCS=( doc/README.html doc/style.css doc/reference/html/. )` in `src_compile`